pyttxt.py

```diff
@@ -0,0 +1,14 @@
+bcainfo=""" BCA course info:
+it is 3 years undergraduate course."""
+with open("BCA.txt", "w") as file:
+     file.write(bcainfo)
+rows={'x':0,'y':0, 'z':0}
+with open('BCA.txt','r') as bca_intro:
+    lines=bca_intro.readlines()

```

Comment:
whyn to use ".readlines()"?

Reply:
.readlines ( ): Reads all the lines and returns them as a string element in a list